core/org.osate.ui/plugin.xml

```diff
@@ -525,6 +525,259 @@ censes only apply to the Third Party Software and not any other portion of this
              </adapt>
           </or>
        </definition>
+
```

Comment:
Shouldn't the simpler definitions be public because they have higher reuse potential?
Should we even distinguish between private and public definitions?

Reply:
Got rid of `.public` and `.private`